dss-api.yml

```diff
@@ -343,6 +343,16 @@ paths:
                 description: Url to send request to when a bundle comes in that matches this query.
                 type: string
                 format: url # Seems that this is possible now because of https://github.com/swagger-api/swagger-core/issues/1045
+              hmac_secret_key:
```

Comment:
Should hmac_secret_key be a required parameter?

Reply:
No, in cases where the user doesn't want to deal with HMAC-based authentication.